### ISSUE TITLE
Change data format of `timeCreated` on `users` table`

### ISSUE THREAD

**Issue by dragonmantank** [phpmentoring/testapp#87]
We are currently using `\DateTime::ISO8601` to populate the `timeCreated` field for users. This needs to be changed to `Y-m-d H:i:s` format, as using the ISO format causes MySQL 5.7 to throw an error about a bad date format. 

Only critical once we decide to upgrade from MySQL 5.5.

**Comment by sprunka** [comment]
This is also used in the ConversationRepository for conversation 'started_at' will it need to be changed there aswell?

**Comment by dragonmantank** [comment]
Yes. Anywhere we are using that will need to be changed since MySQL will throw up there too. I had just noticed it on the Users table.

**Comment by gabidavila** [comment]
I can work on that.

Other changes needs to be made on the tables too, such as NOT NULL fields being filled up empty because we are not validating it on the code.

Also, @dragonmantank can you confirm to me what the production charset for the database and following tables are? We are running on MySQL 5.5, the default is latin1, just so I can test correctly here.

**Comment by ragnaro89** [comment]
Is this still an issue? If so I can tackle this one :)

**Comment by dragonmantank** [comment]
@ragnaro89 Yes, it should still be an issue. Feel free to make the changes and send in a PR :)
